doc/configuration.rst

```diff
@@ -616,6 +616,19 @@ Publishing configuration
 
        confluence_publish_postfix = '-postfix'
 
+    Postfixes can include placeholders. These placeholders are filled using the
+    format method so formatting types can be used.
+
+    Currently supported placeholders
+
+    * {hash} - Create a reproduceable hash given the title and location based 
```

Comment:
While this text is nicely formatted for the raw content, this will not format as expected in the rendered documentation. The trailing lines for the bullet point need to be aligned with the first line's leading content. In addition, it may be nice to keyword the `{hash}` value (if desired). For example:

```
    * ``{hash}`` - Create a reproduceable hash given the title and location based
      from the project root. Using this placeholder provides an option
      ...
```